dev_tools/pypath

```diff
@@ -0,0 +1,19 @@
+# Copyright 2018 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/usr/bin/bash
+
+export PYTHONPATH="$(pwd)"
```

Comment:
This would overwrite the user's existing PYTHONPATH, which could have unwanted side effects. I'd recommend either appending to it (as with the lines below), or a more involved "add if not present" behavior to avoid flooding the PYTHONPATH.

Reply:
Done!